spritz/app/src/main/res/layout/main.xml

```diff
@@ -42,4 +42,24 @@
 
   </LinearLayout>
 
+  <TextView
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:layout_alignParentBottom="true"
+    android:layout_marginBottom="16dp"
+    android:padding="16dp"
```

Comment:
Why not collapsing it in the margin?

```xml


  <TextView
    android:layout_width="wrap_content"
    android:layout_height="wrap_content"
    android:layout_alignParentLeft="true"
    android:layout_alignParentStart="true"
    android:layout_alignParentBottom="true"
    android:layout_marginLeft="16dp"    
    android:layout_marginStart="16dp"
    android:layout_marginBottom="32dp"
    android:text="@string/love_text" />
```

(note that you have `android:supportsRtl="true"` in the manifest and `minSdkVersion` lower than 17 so you need both `left` and `start`) in there.